Resources/Prototypes/Entities/Objects/Tools/tools.yml

```diff
@@ -497,3 +497,27 @@
       Wood: 50
   - type: StaticPrice
     price: 25
+
+- type: entity
+  parent: BaseItem
+  id: RollingPin
+  name: rolling pin
+  description: Used to knock out the Bartender.
+  components:
+  - type: Sprite
+    sprite: Objects/Tools/rolling_pin.rsi
+    state: icon
+  - type: Item
+    sprite: Objects/Tools/rolling_pin.rsi
+    size: 10
+  - type: ItemCooldown
+  - type: MeleeWeapon
+    damage:
+      types:
+        Blunt: 8
```

Comment:
Damage seems excessive.

Reply:
its the same as a crowbar which is still less than any of the chefs knives

Reply:
Please reduce it from the crowbar.

Reply:
7 :trollface: